bundle/manifests/konveyor-operator.clusterserviceversion.yaml

```diff
@@ -169,7 +169,7 @@ spec:
                 - name: RELATED_IMAGE_TACKLE_HUB
                   value: quay.io/konveyor/tackle2-hub:latest
                 - name: RELATED_IMAGE_TACKLE_POSTGRES
-                  value: quay.io/centos7/postgresql-12-centos7:latest
+                  value: quay.io/centos7/postgresql-12-centos7:centos7
```

Comment:
funny tag... why would a centos7 org require centos7 tag.. does centos7 org sometimes push centos8?

Reply:
I think I found the repo corresponding to this image and opened an issue. We'll see what they say...
https://github.com/sclorg/postgresql-container/issues/558